data_structures/binary_tree/binary_tree_traversals.py

```diff
@@ -0,0 +1,149 @@
+class Node:
+    """
+    A Node has data variable and pointers toits left and right nodes.
+    """
+    def __init__(self,data):
+        self.left=None
+        self.right=None
+        self.data=data
+
+def preOrder(root):
+    """
+    PreOrder traversal: visit root node then its left subtree followed by right subtree.
+    """
+    if(root!=None):
+        print(root.data,end=" ")
+        preOrder(root.left)
+        preOrder(root.right)
+        
+def postOrder(root):
+    """
+    PostOrder traversal: visit left subtree followed by right subtree and then root node.
+    """
+    if(root!=None):
+        postOrder(root.left)
+        postOrder(root.right)
+        print(root.data,end=" ")
+        
+def inOrder(root):
+    """
+    InOrder traversal: visit its left subtree followed by root node and then right subtree.
+    """
+    if(root!=None):
+        inOrder(root.left)
+        print(root.data,end=" ") 
+        inOrder(root.right)
+
+def Height(root):
+    """
+    Recursive function for calculating height of the binary tree.
+    """
+    if(root==None):
+        return 0
+    leftHeight=Height(root.left)
+    rightHeight=Height(root.right)
+    if leftHeight>rightHeight:
+        return leftHeight+1
+    else:
+        return rightHeight+1

```

Comment:
```suggestion
    return max(leftHeight, rightHeight) + 1
```

Reply:
i don't think that is required because one of the if-else statement would surely be executing.